docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "3.8"
+
+services:
+  app:
+    build: .
+    command: >
+      sh -c "python manage.py wait_for_db && python manage.py migrate && python manage.py collectstatic --noinput && python manage.py runserver 0.0.0.0:8000"
+    env_file:
+        - .env
+    volumes:
+      - static_volume:/app/static
+      - media_volume:/app/media
+      - staticfiles:/app/staticfiles
+    depends_on:
+      - db
```

Comment:
The `depends_on` option does not wait for the database to be 'ready', only for the container to be started. Consider using a script or tool to check the database readiness before starting the application.